test/relayer/test_relayer.go

```diff
@@ -0,0 +1,193 @@
+package relayer
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/avast/retry-go"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/ory/dockertest"
+	"github.com/ory/dockertest/docker"
+	"github.com/strangelove-ventures/ibc-test-framework/test/chain"
+	"github.com/strangelove-ventures/ibc-test-framework/test/utils"
+	"github.com/stretchr/testify/require"
+)
+
+type TestRelayer struct {
+	T          *testing.T
+	Pool       *dockertest.Pool
+	Repository string
+	Version    string
+	Bin        string
+	RootDir    string
+}
+
+func NewTestRelayer(
+	t *testing.T,
+	pool *dockertest.Pool,
+	repository string,
+	version string,
+	bin string,
+) *TestRelayer {
+	return &TestRelayer{
+		T:          t,
+		Pool:       pool,
+		Repository: repository,
+		Version:    version,
+		Bin:        bin,
+	}
+}
+
+func (r *TestRelayer) Initialize(
+	ctx context.Context, node1 *chain.TestNode, node2 *chain.TestNode,
+) {
+	rootDir, err := utils.CreateTmpDir()
+	require.NoError(r.T, err)
+	r.RootDir = rootDir
+
+	chain1RPCAddress := fmt.Sprintf(
+		"tcp://%s", node1.Container.GetHostPort("26657/tcp"))
+	chain1Config := fmt.Sprintf(`{
+	"chain-id": "%s",
+	"rpc-addr": "%s",
+	"account-prefix": "cosmos",
+	"gas-adjustment": 1.5,
+	"gas-prices": "0.001umuon",
+	"trusting-period": "10m"
+}`, node1.C.ChainID, chain1RPCAddress)
+
+	chain2RPCAddress := fmt.Sprintf(
+		"tcp://%s", node2.Container.GetHostPort("26657/tcp"))
+	chain2Config := fmt.Sprintf(`{
```

Comment:
people would use 

```go
json.Marshal(map[string]interface{}{"chain-id": node1.C.ChainID, ...})
```
or possibly they'd just use the existing type directly if they could and would `json.Marshal` that thing.